jobs/WebUi/PostTest/docker_post_test.groovy

```diff
@@ -0,0 +1,83 @@
+import groovy.transform.Field;

```

Comment:
I think you should have to revisit the docker-post-test part.
here you just "up" the on-web-ui docker with rackhd dockers, no real function test.
so I think we can revisit the post-tests portion in next PR/next stage.